Makefile

```diff
@@ -2,24 +2,66 @@
 # MAINTAINER: Steve Sloka <slokas@upmc.edu>
 # If you update this image please bump the tag value before pushing.
 
-.PHONY: all binary container push clean test
-
 TAG = 1.6
 PREFIX = upmcenterprises
 
+BIN = registry-creds
+ARCH = amd64
+
+# go option
+GO      ?= go
+LDFLAGS := -w
+GOFLAGS := -a -installsuffix cgo
+
+# docker build arguments for internal proxy
+ifneq ($(http_proxy),)
```

Comment:
It is to build the `dockerfile`: it will not build if working behind a corporate proxy.